metricbeat/tests/system/test_golang.py

```diff
@@ -0,0 +1,55 @@
+import os
+import metricbeat
+import unittest
+import time
+
+GOLANG_FIELDS = metricbeat.COMMON_FIELDS + ["golang"]
+
+
+class Test(metricbeat.BaseTest):
+
+    @unittest.skipUnless(metricbeat.INTEGRATION_TESTS, "integration test")
+    def test_stats(self):
+        """
+        prometheus stats test
+        """
+        self.render_config_template(modules=[{
+            "name": "golang",
+            "metricsets": ["heap"],
+            "hosts": self.get_hosts(),
+            "period": "1s"
+        }])
+        proc = self.start_beat(
+            extra_args=[
+                "-httpprof",
+                os.getenv(
+                    'GOLANG_HOST',
+                    'localhost') +
+                ":" +
+                os.getenv(
+                    'GOLANG_PORT',
+                    '6060')])
+
+        time.sleep(3)
```

Comment:
Why is this needed? This mostly leads to flaky tests. Best use some "log_contains" to block here.